src/Umbraco.Cms.Integrations.Crm.Dynamics/App_Plugins/UmbracoCms.Integrations/Crm/Dynamics/Render/DynamicsForm.cshtml

```diff
@@ -0,0 +1,25 @@
+﻿@using Umbraco.Cms.Integrations.Crm.Dynamics.Models.ViewModels;
+
+@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<FormViewModel>
+
+@{
+    var id = Guid.NewGuid();
+}
+
+@if (Model.IFrameEmbedded)
+{
+    <iframe frameBorder="0" style="width:100%" id=@id></iframe>
+    <script>
+        var sandboxSrc = "https://@(Model.Hostname)/t/formsandbox/@(Model.WebsiteId)/@(Model.FormBlockId)?ad="
```

Comment:
Is it right we are referencing a sandbox here?  Seems odd in that we would be rendering forms in production.

Reply:
Rendering code is picked directly from the Dynamics private space as the code to use for embedding.